CHANGELOG.md

```diff
@@ -124,6 +124,8 @@
   [#9737](https://github.com/Kong/kong/pull/9737)
 - Add `key-sets` entity to group and manage `keys`
   [#9737](https://github.com/Kong/kong/pull/9737)
+- Expose postgres connection pool configuration
```

Comment:
Should move to the `Unreleased` section.

Reply:
Moved